cpt-quickstart.rst

```diff
@@ -171,13 +171,13 @@ Create default.local
 ~~~~~~~~~~~~~~~~~~~~
 
 Create ``/etc/cvmfs/default.local`` and open the file for editing.
-Select the desired repositories by setting ``CVMFS_REPOSITORIES=repo1,repo2,...``. For ATLAS, for instance, set
+For the purposes of convenience when running CVMFS commands, you may wish to optionally list repositories you are interested in by setting ``CVMFS_REPOSITORIES=repo1,repo2,...``. For ATLAS, for instance, set
```

Comment:
```suggestion
For the purposes of convenience when running CVMFS client utility commands (e.g. ``cvmfs_talk``, ``cvmfs_config``), you may wish to optionally list repositories you are interested in by setting ``CVMFS_REPOSITORIES=repo1,repo2,...``. This will then run those commands on all listed repositories.
For ATLAS, for instance, set
```